client/utils/__tests__/form-spec.js

```diff
@@ -0,0 +1,85 @@
+import { expect } from 'chai'
+import td from 'testdouble'
+
+import { formApiAdapter } from '../form'
+
+describe('formApiAdapter', () => {
+  const formValues = {}
+  const action = {}
```

Comment:
We should have something better than empty objects for these.  Again, my bad on the original.